src/libraries/Common/tests/System/Security/Cryptography/MLKemTests.cs

```diff
@@ -522,6 +523,339 @@ public static void ImportEncryptedPkcs8PrivateKey_Both_BytePassword()
             }
         }
 
+        [Fact]
+        public static void ImportFromPem_NullSource()
+        {
+            AssertExtensions.Throws<ArgumentNullException>("source", static () => MLKem.ImportFromPem((string)null));
+        }
+
+        [Fact]
+        public static void ImportFromPem_PublicKey_Roundtrip()
+        {
+            foreach ((MLKemAlgorithm algorithm, byte[] spki) in SubjectPublicKeyInfoTestData)
+            {
+                AssertImportFromPem(ImportFromPem =>
+                {
+                    string pem = WritePem("PUBLIC KEY", spki);
```

Comment:
Nit: I think this is the only test that didn't hoist this-equivalent work outside of the callback.  Since `spki` doesn't vary across the overloads, `pem` won't vary across the overloads, so this line can be hoisted.

Reply:
There were two, but yeah makes sense.